tests/user_conf.py

```diff
@@ -0,0 +1,12 @@
+"""
+    Helper file. Contains user configuration for user categories mapping.
+    This file has to be imported with * (sadly)
```

Comment:
because of:
user_definitions = None
if hasattr(settings, "UNIVERSAL_NOTIFICATIONS_USER_DEFINITIONS_FILE") and \
        hasattr(settings, "UNIVERSAL_NOTIFICATIONS_USER_CATEGORIES_MAPPING"):
    user_definitions = importlib.import_module(settings.UNIVERSAL_NOTIFICATIONS_USER_DEFINITIONS_FILE)
well, actually it is not imported with *.
Updated description.